### PR TITLE
feat: add new Jitsu events

### LINKED ISSUE #6968: [CAL-1024] new jitsu tracking: website_view, signup, onboarding_finished, team_created
see previous jitsu.track functions for reference

`website_view` would need to be tracked in calcom/website repo

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-1024](https://linear.app/calcom/issue/CAL-1024/new-jitsu-tracking-website-view-signup-onboarding-finished-team)</sub>

### PR DESCRIPTION
**Pull request by G3root**
## What does this PR do?

Fixes #6968

**Environment**: Staging(main branch) / Production

## Type of change

- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Review by PeerRich — APPROVED** [review]

File: apps/web/pages/signup.tsx [hunk 1]
```diff
@@ -56,7 +56,7 @@ export default function Signup({ prepopulateFormValues }: inferSSRProps<typeof g
     })
       .then(handleErrors)
       .then(async () => {
-        telemetry.event(telemetryEventTypes.login, collectPageParameters());
+        telemetry.event(telemetryEventTypes.signup, collectPageParameters());
```
[the remark this answers is not in the document]
Comment (PeerRich) [reply apps/web/pages/signup.tsx:59]: actually this signup component is only used for self-hosters

**Comment by PeerRich** [comment]
can we do "onboarding_started"  too?

**Comment by G3root** [comment]
@PeerRich added for `onboarding_started` as well

File: apps/web/pages/auth/login.tsx [hunk 2]
```diff
@@ -182,8 +182,6 @@ export default function Login({
                     StartIcon={FaGoogle}
                     onClick={async (e) => {
                       e.preventDefault();
-                      // track Google logins. Without personal data/payload
-                      telemetry.event(telemetryEventTypes.googleLogin, collectPageParameters());
```
Comment (PeerRich) [inline apps/web/pages/auth/login.tsx:186]: @G3root for your info, i'm using this opportunity to remove SAML and Google Sign in tracking. It doesn't provide any useful insights and will lower our tracked events per month

File: packages/features/auth/SAMLLogin.tsx [hunk 3]
```diff
@@ -43,9 +41,6 @@ export function SAMLLogin({ samlTenantID, samlProductID, setErrorMessage }: Prop
       onClick={async (event) => {
         event.preventDefault();
 
-        // track Google logins. Without personal data/payload
-        telemetry.event(telemetryEventTypes.googleLogin, collectPageParameters());
-
         if (!HOSTED_CAL_FEATURES) {
```
Comment (PeerRich) [inline packages/features/auth/SAMLLogin.tsx:44]: same here

File: packages/features/ee/sso/components/OIDCConnection.tsx [hunk 4]
```diff
@@ -56,12 +55,10 @@ const CreateConnectionDialog = ({
 }) => {
   const { t } = useLocale();
   const utils = trpc.useContext();
-  const telemetry = useTelemetry();
   const form = useForm<FormValues>();
 
   const mutation = trpc.viewer.saml.updateOIDC.useMutation({
     async onSuccess() {
-      telemetry.event(telemetryEventTypes.samlConfig, collectPageParameters());
```
Comment (PeerRich) [inline packages/features/ee/sso/components/OIDCConnection.tsx:64]: same

File: packages/features/ee/sso/components/SAMLConnection.tsx [hunk 5]
```diff
@@ -54,12 +53,10 @@ const CreateConnectionDialog = ({
 }) => {
   const { t } = useLocale();
   const utils = trpc.useContext();
-  const telemetry = useTelemetry();
   const form = useForm<FormValues>();
 
   const mutation = trpc.viewer.saml.update.useMutation({
     async onSuccess() {
-      telemetry.event(telemetryEventTypes.samlConfig, collectPageParameters());
```
Comment (PeerRich) [inline packages/features/ee/sso/components/SAMLConnection.tsx:62]: same

**Pull request merged**